values.yaml

```diff
@@ -852,7 +852,7 @@ trivy:
   # Anonymous downloads from GitHub are subject to the limit of 60 requests per hour. Normally such rate limit is enough
   # for production operations. If, for any reason, it's not enough, you could increase the rate limit to 5000
   # requests per hour by specifying the GitHub access token. For more details on GitHub rate limiting please consult
-  # https://developer.github.com/v3/#rate-limiting
+  # https://v2.12.0eloper.github.com/v3/#rate-limiting
```

Comment:
Wrong replacement?

Reply:
Good catch! Thanks ! And I will fix in the next release